### DISCUSSION TITLE
How to recover deleted libc.so.6

### DISCUSSION THREAD

**Discussion by fishjojo** [microsoft/WSL#10251] [Q&A]
I accidentally removed the `libc.so.6` file, and now I'm not able to start wsl. How can I fix this?
I tried to add back the symbolic link in powershell with the command `New-Item -ItemType SymbolicLink`, but it didn't allow me to do this.
Thank you in advance.

**Comment by teoparma** [comment]
I had the same problem, I resolved it thanks to NotTheDr01ds' aswer at this question: https://askubuntu.com/questions/1458921/bash-error-and-wsl-not-working
